components/using_components.rst

```diff
@@ -40,7 +40,7 @@ If you know you need a specific version of the library, add that to the command:
 
 .. code-block:: bash
 
-    $ composer require symfony/finder
+    $ composer require symfony/finder 2.6.2
```

Comment:
-1. You should not use an exact match in your composer constraint (it means composer will never give you bug fixes until you ask for them explicitly). And Composer is smart enough to guess the best constraint based on the existing versions of the package. So most of the time, the best solution is to omit the version constraint entirely